flocker/node/agents/functional/test_cinder.py

```diff
@@ -71,14 +69,17 @@ class CinderBlockDeviceAPIInterfaceTests(
     Block devices that are created in these tests will be cleaned up by
     ``TidyCinderVolumeManager``.
     """
+    def test_foreign_cluster_volume(self):
+        """
+        Volumes from other Flocker clusters are not listed.
+        """
+        block_device_api2 = cinderblockdeviceapi_for_test(
+            test_case=self,
+            cluster_id=uuid4(),
+        )
 
+        self.assertVolumesDistinct(block_device_api2)
```

Comment:
We could just add them to a mixin class and mix that into the Cinder and EBS tests for now...and figure our if / how to run them against the LoopbackBlockDeviceAPI later.